src/go/service/sctpd/sctpd.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2021 The Magma Authors.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package sctpd contains implementations of magma.sctpd.SctpdDownlink and
```

Comment:
is the plan to fill this out or is this for package level comments?

Reply:
currently just for package level comments; there might be some code in here at some point.

I looked around the base libs and there aren't really any `doc.go` files, even though that appears to be popular in some libraries. Most of the time, the package level comment is in the file that matches the package name, but sometimes it's in a random file (which seems bad to me).